src/persist/src/cfg.rs

```diff
@@ -144,6 +145,8 @@ impl ConsensusConfig {
     pub fn try_from(
         value: &str,
         connection_pool_max_size: usize,
+        connection_pool_ttl: Duration,
```

Comment:
this is starting to get unwieldy. can we pull out a `trait ConsensusKnobs` that is defined in mz_persist (not mz_persist_config) and that PersistConfig implements, please? that will also set us up nicely for making any dynamic config changes be reflected immedately

Reply:
done! v excited about setting things up for dynamic config